test/test_dependencies.jl

```diff
@@ -0,0 +1,47 @@
+using Mimi
+
+#list of URLs of branches of packages to test
+dependencies = ["https://github.com/davidanthoff/fund.jl/archive/master.zip"]
+
+#list of failed tests to build as you go
+errors = []
+
+#make a temporary directory to run the tests in
+run(`mkdir tmp_testing`)
+cd("tmp_testing")
+
+#loop through each dependent package
+for url in dependencies
+  #download the package
+  run(`wget $url`)
+  zip_name = chomp(readall(`basename $url`))
```

Comment:
Maybe https://github.com/JuliaWeb/URIParser.jl could be used here so that this works cross-platform?